iptool_test.go

```diff
@@ -5,21 +5,18 @@ import (
 	"testing"
 
 	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
 )
 
 func TestIsPrivate(t *testing.T) {
-	tool, err := New()
-	if !assert.NoError(t, err) {
-		return
-	}
+	tool, includesLocalInterfaces := New()
+	require.True(t, includesLocalInterfaces)
```

Comment:
Im not sure with replacing `assert` with `require` if we keep require then if an error fails the test is gonna be skipped, contrary to assert which logs each test and if fails stops on the one that failed. But please feel free to correct me if im wrong or it's behavior is different.

Reply:
Yeah, this would actually be a good place for an assert. I'll change, thanks!

I tend to use require when checking for errors, since there's little point in continuing if we encountered an error.

Reply:
Oh actually, now that I think about it, if includesLocalInterfaces is false, there are later test conditions that will also fail because the test assumes that local interfaces are being included. So, I'll stick with `require` here :)